src/SwaggerProvider.DesignTime/OperationCompiler.fs

```diff
@@ -62,13 +71,13 @@ type OperationCompiler (schema:SwaggerObject, defCompiler:DefinitionCompiler) =
             // Fit headers into quotation
             let headers =
                 let jsonConsumable = op.Consumes |> Seq.exists (fun mt -> mt="application/json")
-                <@@
+                <@
```

Comment:
generally got away from the untyped expression API as much as possible in this PR.